content/pt/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md

```diff
@@ -0,0 +1,129 @@
+---
+reviewers:
+- femrtnz
+- jcjesus
+title: Adicionando entradas ao Pod /etc/hosts com o HostAliases
+content_template: templates/concept
+weight: 60
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+Adicionar entradas ao arquivo /etc/hosts de um Pod fornece uma substituição de nível de host da resolução do nome do host quando o DNS e outras opções não são aplicáveis. Na versão 1.7, os usuários podem adicionar essas entradas personalizadas ao campo HostAliases no PodSpec.
+
+A modificação que não usa o HostAliases não é sugerida porque o arquivo é gerenciado pelo Kubelet e pode ser sobrescrito durante a criação ou reinicialização do Pod.
```

Comment:
what do you think about invert to "Não é sugerida a modificação que não usa o HostAliases porque..."